blocks/video-promo-block/features/video-promo/default.jsx

```diff
@@ -18,36 +20,58 @@ const DescriptionText = styled.p`
 `;
 
 const AlertBadge = styled.span`
+  background-color: #db0a07;
+  border-radius: 5px;
+  color: #fff;
   display: inline-block;
   padding: 0.3rem 0.8rem;
-  color: #FFFFFF;
-  background-color: #DB0A07;
-  border-radius: 5px;
 `;
 
 const VideoPromo = ({ customFields }) => {
   const { arcSite } = useFusionContext();
   const {
-    uuid,
     autoplay = false,
-    ratio = 0.5625,
-    title,
-    description,
+    inheritGlobalContent = true,
     playthrough = false,
```

Comment:
playthrough looks like it's default is already false based on default proptypes. you can remove the = false
```suggestion
    playthrough,
```
![Screen Shot 2020-11-19 at 14 22 41](https://user-images.githubusercontent.com/5950956/99719952-a05cb280-2a72-11eb-8e72-91a6eb778db7.png)